packages/x-charts/src/RadarChart/RadarSeriesPlot/RadarSeriesMarks.tsx

```diff
@@ -2,28 +2,40 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import { useRadarSeriesData } from './useRadarSeriesData';
 import { RadarSeriesMarksProps } from './RadarSeriesPlot.types';
+import { useItemHighlightedGetter } from '../../hooks/useItemHighlightedGetter';
 
 function RadarSeriesMarks(props: RadarSeriesMarksProps) {
   const { seriesId, ...other } = props;
   const seriesCoordinates = useRadarSeriesData(props.seriesId);
 
+  const { isFaded, isHighlighted } = useItemHighlightedGetter();
+
   return (
     <React.Fragment>
-      {seriesCoordinates?.map(({ seriesId: id, points, color }) => (
-        <g key={id}>
-          {points.map((point, index) => (
-            <circle
-              key={index}
-              cx={point.x}
-              cy={point.y}
-              r={5}
-              fill={color}
-              stroke={color}
-              {...other}
-            />
-          ))}
-        </g>
-      ))}
+      {seriesCoordinates?.map(({ seriesId: id, points, color, hideMark, fillArea }) => {
+        if (hideMark) {
+          return null;
+        }
+        const isItemHighlighted = isHighlighted({ seriesId: id });
+        const isItemFaded = !isItemHighlighted && isFaded({ seriesId: id });
+
+        return (
+          <g key={id}>
+            {points.map((point, index) => (
+              <circle
+                key={index}
+                cx={point.x}
+                cy={point.y}
+                r={3}
+                fill={color}
+                stroke={color}
+                opacity={fillArea && isItemFaded ? 0.5 : 1}
```

Comment:
@noraleonte In addition to your recomendations, I added this opacity rule for fadded area. Otherwise marks on to of the highlighted area were too visible